docker/puppet-bolt/spec/dockerfile_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rspec/core'
+
+SPEC_DIRECTORY = File.dirname(__FILE__)
+
+describe 'puppet-bolt container' do
+  include Pupperware::SpecHelpers
+
+  before(:all) do
+    @image = ENV['PUPPET_TEST_DOCKER_IMAGE']
+    if @image.nil?
+      error_message = <<-MSG
+* * * * *
+  PUPPET_TEST_DOCKER_IMAGE environment variable must be set so we
+  know which image to test against!
+* * * * *
+      MSG
+      fail error_message
+    end
+  end
+
+  it 'should run the image' do
+    result = run_command("docker run --detach #{@image}")
+    container = result[:stdout].chomp
+    wait_on_container_exit(container)
+    expect(get_container_exit_code(container)).to eq(0)
+    emit_log(container)
+    teardown_container(container)
+  end
+
+  it 'should run a bolt command and analytics should be disabled' do
+    result = run_command("docker run -i #{@image} command run whoami -t localhost")
+    expect(result[:stdout]).to match(/root/)
+    expect(result[:stdout]).to match(/Analytics opt-out is set, analytics will be disabled/)
```

Comment:
This check is giving me trouble. Even if I add `--debug` to the bolt command (which I think we need) it looks like debug statements get sent to a separate stream or something? It's not in the result hash at all.

Reply:
how are you invoking the tests?

Reply:
Yeah that makes sense, log messages will be on stderr.

Reply:
Oh, jeeze yeah i think that is printed to stderr will look in to that helper

Reply:
lol
```
result = run_command("docker run -i #{@image} command run whoami -t localhost --debug 2>&1")
```
works